### ISSUE TITLE
'refresh' Command Exception

### ISSUE THREAD

**Issue by D1G1T4L3CH0** [DeathCradle/Terraria-s-Dedicated-Server-Mod#122]
Using the `refresh` command with non-op player throws an exception.
```
7/7/2015 3:31:19 PM Pool> System.Collections.Generic.KeyNotFoundException: LastCostlyCommand
  at System.Collections.Concurrent.ConcurrentDictionary`2[System.String,System.Object].GetValue (System.String key) [0x00000] in <filename unknown>:0 
  at System.Collections.Concurrent.ConcurrentDictionary`2[System.String,System.Object].get_Item (System.String key) [0x00000] in <filename unknown>:0 
  at tdsm.core.PlayerExtensions.GetLastCostlyCommand (Terraria.Player player) [0x00000] in <filename unknown>:0 
  at tdsm.core.Entry.Refresh (ISender sender, tdsm.api.Command.ArgumentList args) [0x00000] in <filename unknown>:0 
  at tdsm.api.Command.CommandInfo.Run (ISender sender, tdsm.api.Command.ArgumentList args) [0x00000] in <filename unknown>:0 
  at tdsm.api.Command.CommandParser.ParseAndProcess (ISender sender, System.String line) [0x00000] in <filename unknown>:0 
  at tdsm.api.Command.CommandParser.ParsePlayerCommand (Terraria.Player player, System.String line) [0x00000] in <filename unknown>:0 
  at tdsm.core.Entry.OnChat (tdsm.api.Plugin.HookContext& ctx, tdsm.api.Plugin.PlayerChat& args) [0x00000] in <filename unknown>:0 
  at tdsm.api.Plugin.HookPoint`1[tdsm.api.Plugin.HookArgs+PlayerChat].Invoke (tdsm.api.Plugin.HookContext& context, tdsm.api.Plugin.PlayerChat& arg) [0x00000] in <filename unknown>:0 
```

**Comment by DeathCradle** [comment]
This should fix it, 287ad426a221a5779146550a4b06183097a3bc0f

**Comment by D1G1T4L3CH0** [comment]
Resolved.
